microsoft-edge/extensions-chromium/developer-guide/port-chrome-extension.md

```diff
@@ -32,7 +32,7 @@ To port your Chrome extension:
 <!-- ====================================================================== -->
 ## Setting allowed_origins for a native app
 
-If your extension exchanges messages with a native app using `chrome.runtime.connectNative`, make sure that you set `allowed_origins` to `extension://[Microsoft-Catalog-extensionID]` in your native messaging host manifest file.  The setting allows the app to identify your extension.
+If your extension exchanges messages with a native app using `chrome.runtime.connectNative`, make sure that you set `allowed_origins` to `chrome-extension://[Microsoft-Catalog-extensionID]` in your native messaging host manifest file.  The setting allows the app to identify your extension.
 
 
```

Comment:
```suggestion
Example native messaging host manifest file: 

```json
{
   "name": "com.my_company.my_app",
   "description": "My App",
   "path": "C:\Program Files\My App\chrome_native_messaging_host.exe",
   "type": "stdio",
   "allowed_origins": [
      "chrome-extension://[Microsoft-Catalog-extensionID]/"
   ]
}
```

```
Add this listing, from the Issue?  @sonakshimore @vchapel